src/scss/customOverrides.scss

```diff
@@ -0,0 +1,3 @@
+@import "scss/constants.scss";
+
+// Custom CSS file to deal with specific client requests
```

Comment:
```suggestion
// Custom CSS file to deal with environment specific hacks/overrides
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1647/commits/2b61b8cc3f4f16c3310a72d0e2059b7c070d091f